sklearn/svm/base.py

```diff
@@ -571,7 +573,17 @@ def predict(self, X):
         y_pred : array, shape (n_samples,)
             Class labels for samples in X.
         """
-        y = super().predict(X)
+        check_is_fitted(self, "classes_")
+        if self.break_ties and self.decision_function_shape == 'ovo':
+            raise ValueError("break_ties can only be True if "
+                             "decision_function_shape is 'ovo'")
```

Comment:
I read this as "break_ties *must be True* when decision_function_shape is ovo", which is not the intended purpose

(Also I think you meant ovr?)

Maybe

> break_ties must be False when decision_function_shape is 'ovo'

or

> break_ties can only be specified when decision_function_shape is 'ovr'